src/database/queries/changemakers/updateById.sql

```diff
@@ -1,4 +1,9 @@
 UPDATE changemakers
-SET keycloak_organization_id = :keycloakOrganizationId
+SET
+	tax_id = coalesce(:taxId, tax_id),
+	name = coalesce(:name, name),
+	keycloak_organization_id = coalesce(
```

Comment:
An unfortunate side effect of the intersection of `coalesce` and the rest of our code is that we cannot nullify an already-set keycloak organization ID here. If we literally pass `null` as the desired ID, `coalesce` leaves the existing UUID. We can't distinguish "I'm setting this as `null`" from "I'm not setting this at all."

Reply:
I think this is something to be worked out later, though, as we have a workaround where it is a limited use case where an organization gets removed and if needed we could set the keycloak_organization_id to an all-zeroes UUID.

Created #1472 to track.